source/_components/http.markdown

```diff
@@ -27,27 +30,26 @@ http:
   api_password: YOUR_PASSWORD
 ```
 
-Configuration variables:
-
-{% configuration http %}
+{% configuration %}
 api_password:
   description: Protect Home Assistant with a password.
   required: false
   type: string
 server_host:
-  description: 'Only listen to incoming requests on specific IP/host (default: bind to `0.0.0.0` which means accept all IPv4 connections). Use `server_host: "::0"` if you want to listen to (and only) IPv6.'
+  description: "Only listen to incoming requests on specific IP/host. By default it will accept all IPv4 connections. Use `server_host: ::0` if you want to listen to (and only) IPv6."
   required: false
   type: string
+  default: 0.0.0.0
 server_port:
   description: Let you set a port to use.
   required: false
-  type: integer
+  type: int
   default: 8123
 base_url:
-  description: "The URL that Home Assistant is available on the internet. For example: `hass-example.duckdns.org:8123`. Defaults to the local IP address. The iOS app finds local installations, if you have an outside URL use this so that you can auto-fill when discovered in the app."
+  description: "The URL that Home Assistant is available on the internet. For example: `hass-example.duckdns.org:8123`. The iOS app finds local installations, if you have an outside URL use this so that you can auto-fill when discovered in the app."
   required: false
   type: string
-  default: The local IP address
+  default: "*the local IP address*"
```

Comment:
Defaults are already rendered as italic, adding some extra is not going to change a thing.